cli/pause.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// pauseDescription is used to describe pause command in detail and auto generate command doc.
+var pauseDescription = "Pause a running container object in Pouchd. " +
+	"when pausing, the container will stop its running but hold all the relevant resource." +
+	"This is useful when you wish to pause a container for a while and to restore the running status later." +
+	"The container you paused will stop running."
+
+// PauseCommand use to implement 'pause' command, it pauses a container.
+type PauseCommand struct {
+	baseCommand
+}
+
+// Init initialize pause command.
+func (p *PauseCommand) Init(c *Cli) {
+	p.cli = c
+	p.cmd = &cobra.Command{
+		Use:   "pause [container]",
+		Short: "Pause a running container",
+		Long:  pauseDescription,
+		//TODO:numbers
+		Args: cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return p.runPause(args)
+		},
+		Example: pauseExample(),
+	}
+	p.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (p *PauseCommand) addFlags() {
+	// TODO: add flags here
+}
+
+// runPause is the entry of pause command.
+func (p *PauseCommand) runPause(args []string) error {
+	apiClient := p.cli.Client()
+
+	container := args[0]
+
+	if err := apiClient.ContainerPause(container); err != nil {
+		return fmt.Errorf("failed to pause container %s: %v", container, err)
+	}
+	return nil
+}
+
+// pauseExample shows examples in pause command, and is used in auto-generated cli docs.
+func pauseExample() string {
+	return `$ pouch ps
+Name     ID       Status    Image
+foo      71b9c1   Running   docker.io/library/busybox:latest
+$ pouch pause foo
+$ pouch ps
+Name     ID       Status    Image
```

Comment:
And if you rebased to master, you will find that in the row, there will be another field named by `Runtime` which is introduced in https://github.com/alibaba/pouch/pull/294.